### PR TITLE
Feature/lobster version flag for tools

### PR DESCRIPTION
**Pull request by kedarnn**

### CODE REVIEW

File: lobster/version.py [hunk 1]
```diff
@@ -25,3 +27,42 @@
...
+    """
+    This decorator function is used on function wherever we are parsing
+    the command line arguments which then adds a version argument to the function.
+    If a version flag is passed to the command line arguments then the LOBSTER version
+    is printed.
+    Parameters
+    ----------
+    obj - obj can be an ArgumentParser object or a Function object.
+
+    Returns - None
```
Comment (mugdhadhole1) [inline lobster/version.py:42]:
```suggestion
    Returns - Nothing
```

**Review by mugdhadhole1 — CHANGES_REQUESTED** [review]
`-v` and `--version`
command runs but the tool version is not printed as output

**Review by mugdhadhole1 — CHANGES_REQUESTED** [review]
While I check the version for Lobster-cpp tool, it does show the version, but it asks for mandatory arguments as input to the tool

**Comment by kedarnn** [comment]
Have added support for all the missing tools.

**Review by phiwuu — COMMENTED** [review]
Please update the `CHANGELOG.md`

**Review by mugdhadhole1 — APPROVED** [review]

**Pull request merged**